config/config.js

```diff
@@ -192,6 +192,18 @@ var initGlobalConfig = function () {
   // to avoid running test suites on a prod/dev environment (which delete records and make modifications)
   if (process.env.NODE_ENV !== 'test') {
     config = _.merge(config, (fs.existsSync(path.join(process.cwd(), 'config/env/local.js')) && require(path.join(process.cwd(), 'config/env/local.js'))) || {});
+  } else {
+    // Check if we have a local.js custom/local environment. If one exists, we need to check for a test database setting 
+    // that will override the current config object's "db" field. Given the fact that newer versions of MongoDB are 
+    // deprecating the MONGODB-CR authentication. In such cases, mongoose won't connect without the "user" & "pass" options. 
```

Comment:
@mleanos I don't see why that wouldn't be something that we could ask instead of all this. I get what this is doing now, but I'm really not fond of how we go about it, and I think it's too much of a stretch when you can just avoid the issue.